data-estate-connecter/shortcodes/shortcodes.php

```diff
@@ -536,29 +573,86 @@ function dec_txa_button($atts,$content=null) {
 	//$wsdl="https://www.au.v3travel.com/CABS.WebServices/SearchService.asmx?WSDL";
 
 }
+
+/***Shortcode For Event Date  ****/
 function dec_event_date($atts, $content=null) {
-	extract(shortcode_atts(array('format'=>'l, jS F Y', 'asEndDate'=>'false'), $atts));
+	extract(shortcode_atts(array('type' => 'list','format'=>'l, jS F Y', 'asEndDate'=>'false'), $atts));
 	global $api_arry;
 	$error= api_error_function();
 	if($error){
 		return $error;
 	}
 	else {
-		if ($asEndDate!='false') {
-			if (isset($api_arry->{'latest_end_date'})) {
-				$date = new DateTime($api_arry->{'latest_end_date'});
-				return $date->format($format);
+		if ($type == 'list') {
+			$result_string="<ul class='dec-date'>";
+			foreach ($api_arry->{'info'}->{'dates'} as $event_date) {
+				$sdate = new DateTime($event_date->{'start_time'});
+				$edate = new DateTime($event_date->{'end_time'});
+				$stime_format = $sdate->format("H:i");
+				$etime_format = $edate->format("H:i");
+				if ($stime_format == "00:00" && $etime_format == "00:00") {
+					$sdate_str = $sdate->format("D, d/m/y");

```

Comment:
Suggest declaring "D, d/m/y" as a variable, so that we don't have to type it twice? Do we need to update the default format value above as well (l, jS F Y'), or can we reuse that?